tests/iele/unit/fptr.iele

```diff
@@ -27,6 +27,8 @@ contract test {
     revert -1
   }
 
+  @unused = 13
+
```

Comment:
Why was that added?

Reply:
Testing that we still compute the right offset for test2